msteams-platform/sbs-gs-notificationbot.yml

```diff
@@ -117,7 +117,7 @@ items:
         :::image type="content" source="~/assets/images/teams-toolkit-v2/prerequisites/upload-custom-app.png" alt-text="Illustration shows the option to upload a custom app in Teams.":::
 
         > [!NOTE]
-        > Contact Teams administrator, if you don't have the option to upload a custom app.
+        > Contact Teams administrator, if you don't find the option to upload a custom app.
```

Comment:
Check what version is appropriate and retain that. 

If the so-called option is not available, then the original text is correct.
If the option cannot be found or is hidden, then the latter text is correct (with a minor update that if you can't find the option...)